recipes/ggcaller/meta.yaml

```diff
@@ -1,46 +1,62 @@
 {% set name = "ggCaller" %}
-{% set version = "1.2.4" %}
+{% set version = "1.3.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/samhorsfield96/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: 6438f18d3c99e33aa810e066dd0e175c573bbd6e1324893f13c79e5ae0fdf8d2
+  sha256: 2dedf76907601769be796ebc7c02291b6bcab4892b1a7e5a2c45bc3d1812ce30
 
 build:
-  number: 2
-  skip: True  # [osx or py < 38]
-  script: {{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt
+  number: 0
+  skip: True  # [py < 38]
+  script: CPLUS_INCLUDE_PATH=$PREFIX/include {{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt
   entry_points:
     - ggcaller = ggCaller.__main__:main
 
 requirements:
   build:
     - {{ compiler('c')}}
     - {{ compiler('cxx') }}
-    - cmake >=3.4
+    - cmake >=3.10
     - make
-    - setuptools
-    - llvm-openmp  # [osx]
-    - libgomp      # [linux64]
+    - tbb
+    - tbb-devel
   host:
     - python
     - pybind11
-    - seqan3 <3.1
+    - eigen 3.3
     - bifrost
-    - pthread-stubs
     - zlib
+    - pip
     - bzip2
+    - pytorch
```

Comment:
Just double-checking this is needed for build?

Reply:
This is required for libtorch linking in CMakeLists.txt.